ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -38,40 +28,228 @@ module('Acceptance | kv-v2 workflow | secret and version create', function (hook
 
   module('admin persona', function (hooks) {
     hooks.beforeEach(async function () {
-      const token = await runCmd([
-        createPolicyCmd('admin', adminPolicy(this.backend)),
-        createTokenCmd('admin'),
-      ]);
+      const token = await runCmd(tokenWithPolicyCmd('admin', personas.admin(this.backend)));
       await authPage.login(token);
     });
-    test.skip('create new root secret flow', async function (assert) {
-      assert.expect(0);
-      // TODO: navigate to kv backend from backends
-      // ✅ shows list items / no empty state
-      // ✅ shows correct toolbar options
-      // click "Create secret"
-      // ✅ URL is vault/secrets/:backend/kv/create
-      // ✅ Path input is empty
-      // click [data-test-kv-save]
-      // ✅ Shows validation errors
+    test('create & update root secret with default metadata', async function (assert) {
+      const backend = this.backend;
+      const secretPath = 'some secret';
+      await visit(`/vault/secrets/${backend}/kv/list`);
+      await click(PAGE.list.createSecret);
+
+      // Create secret form -- validations
+      await click(FORM.saveBtn);
+      assert.dom(FORM.invalidFormAlert).hasText('There is an error with this form.');
+      assert.dom(FORM.validation('path')).hasText("Path can't be blank.");
+      await typeIn(FORM.inputByAttr('path'), secretPath);
+      // TODO: clear invalid form alert on form change?
```

Comment:
I think you added this functionality above? 

I originally hadn't cleared it because `AlertInline` adds a loading icon that when a form is re-submitted. I think that UX is preferred over clearing validations/errors on form change (unless the validation corresponds to an input that is fixed) 
![alert](https://github.com/hashicorp/vault/assets/68122737/b7a6ae64-a0bc-48eb-accf-0edaf5eecfa1)